app/services/analytics_events.rb

```diff
@@ -11,6 +11,46 @@
 #                 ||     ||
 
 module AnalyticsEvents
+  # @param [Boolean] success Check whether threatmetrix succeeded properly.
+  # @param [String] transaction_id Vendor-specific transaction ID for the request.
+  # @param [String, nil] client Client user was directed from when creating account
+  # @param [array<String>, nil] errors error response from api call
+  # @param [String, nil] exception Error exception from api call
+  # @param [Boolean] timed_out set whether api call timed out
+  # @param [String] review_status TMX decision on the user
+  # @param [String] account_lex_id LexID associated with the response.
+  # @param [String] session_id Session ID associated with response
+  # @param [Hash] response_body total response body for api call
+  # Result when threatmetrix is completed for account creation and result
+  def account_creation_tmx_result(
```

Comment:
Minor: We use "account_creation" naming here, but the configuration name uses "authentication". I think we should align these. I prefer the former, since we're not talking about the sign-in flow.

Reply:
I was thinking we would have the same policy for account creation and authentication for when we move to use both. meanwhile for the result it will be different depending on how the user goes through it.

Reply:
I'm not sure there are plans to expand this to authentication.